src/Routing/Router.php

```diff
@@ -978,13 +1005,82 @@ public static function scope($path, $params = [], $callback = null)
         $options = [];
         if (is_array($params)) {
             $options = $params;
-            unset($params['routeClass'], $params['extensions']);
+            $cache = (string)Hash::get($params, '_cache');
+            unset($params['routeClass'], $params['extensions'], $params['_cache']);
+            if (!empty($cache) && static::_isCacheSerialize($cache)) {
```

Comment:
code fixed to make it simpler (hopefully) to understand. I can't return at that point because the scope function returns void and I would like to keep it BC